.github/workflows/ci.yml

```diff
@@ -96,43 +124,37 @@ jobs:
         with:
           path: target
           # The target directory is only useful with the same Rust version, dependencies and operating system.
-          key: ${{ matrix.os }}-target-${{ steps.toolchain.outputs.rustc_hash }}-${{ hashFiles('Cargo.lock') }}-build-v3
-
-      - name: Build ${{ matrix.os }} binary
-        run: make build
+          key: ${{ matrix.target }}-target-${{ steps.toolchain.outputs.rustc_hash }}-${{ hashFiles('Cargo.lock') }}-build-v3
 
       - name: Build docs
-        run: make doc
-
-      - name: Run unit tests (no docker)
-        if: matrix.os != 'ubuntu-latest'
-        run: make test
+        run: cargo doc
 
-      - name: Run unit tests (with docker)
-        if: matrix.os == 'ubuntu-latest'
-        run: make test_docker
+      - name: Run unit tests
+        if: matrix.target != 'armv7-unknown-linux-gnueabihf' # We don't want to run tests for ARM as we are only cross-compiling
```

Comment:
That could be an option in the matrix actually. The idea being "run only test only once per platform".

Reply:
I can make another entry in the matrix yeah. Wasn't sure if it is worth it :)